### PR TITLE
Set the "no metricbeat data" alert throttle period correctly

### PR DESCRIPTION
**Pull request by Crazybus**
This used to be defaulting to what was in template.py instead of
correctly being updated to the `throttle` value in `kuberwatcher.yml`

### CODE REVIEW

File: kuberwatcher.py [hunk 1]
```diff
@@ -245,7 +245,7 @@ def main(es, defaults):
     namespaces = get_namespaces(defaults)
     pods = get_all_pods(namespaces)
     watches = generate_watch(pods)
-    watches['metricbeat'] = add_alerts(metricbeat_template, defaults['alerts'], defaults.get('reply_to', None))
+    watches['metricbeat'] = add_alerts(metricbeat_template, defaults['alerts'], defaults.get('throttle', 3600000), defaults.get('reply_to', None))
     watches['metricbeat']['metadata']['message'] = 'No metricbeat data has been recieved in the last 5 minutes! <{0}|kibana>'.format(defaults['kibana_url'])
```
Comment (fatmcgav) [inline kuberwatcher.py:249]: `last 5 minutes!` is no longer accurate? 

https://humanfriendly.readthedocs.io/en/latest/api.html#humanfriendly.format_timespan Could be quite cool, if pulling in another lib isn't an issue :)

**Review by fatmcgav — CHANGES_REQUESTED** [review]
One comment around the `message` value. 

Otherwise, changes all look good :)

**Comment by fatmcgav** [comment]
Also, does this need some new tests to explicitly override the `throttle` config item?

File: kuberwatcher.py (hunk 1 above)
Comment (fatmcgav) [reply kuberwatcher.py:249]: Ah, ignore that comment... Confusing `throttle` with `window`... :doh:

Comment (fatmcgav) [reply kuberwatcher.py:249]: Actually, I think there's still something there... 

Does overriding the `watcher.window` item actually change this `message`?

Comment (Crazybus) [reply kuberwatcher.py:249]: > Does overriding the watcher.window item actually change this message?

 Nope, but I'm working on another PR to make sure that the window itself is actually configurable for the "no metricbeat data" alert and that the text is also updated properly ( including other useful optional metadata like cluster name).

**Comment by Crazybus** [comment]
> Also, does this need some new tests to explicitly override the throttle config item?

Great idea, adding an extra couple of tests now.

File: kuberwatcher.py (hunk 1 above)
Comment (fatmcgav) [reply kuberwatcher.py:249]: Cool, ok :)

**Comment by fatmcgav** [comment]
> > Also, does this need some new tests to explicitly override the throttle config item?
> 
> Great idea, adding an extra couple of tests now.

Cool, ok... I'll hold off approving just yet then...

**Review by fatmcgav — APPROVED** [review]
LGTM

**Pull request merged**